app/Models/Asset.php

```diff
@@ -334,6 +336,59 @@ public function checkOut($target, $admin = null, $checkout_at = null, $expected_
         return false;
     }
 
+    /**
+     * Checks the asset in to the target
+     *
+     * @TODO I removed the admin variable when writing this, because it is still unused down here too. Does it need to be added again?
+     *
+     * @author [A. Janes] [<ajanes@adagiohealth.org>]
+     * @param Carbon $checkin_at
+     * @param string $note
+     * @param null $name
```

Comment:
I included the `$name` variable to keep it consistent with `checkOut()`. It isn't going to be used by this PR. But, it seemed like it made sense to keep if this method is reused.